addons/account_test/account_test_data.xml

```diff
@@ -151,5 +150,4 @@ if result:
   result.insert(0,_('* Here is the list of inactive partners and movement lines that are not correct: '))
```

Comment:
Please change the test data according to model changes.
Currently it give's error while printing reports .